pennylane/operation.py

```diff
@@ -964,6 +964,77 @@ def kraus_matrices(self):
         return self._kraus_matrices(*self.parameters)
```

Comment:
Worth adding to the docstrings at the top of the file? These actually are rendered in the main docs:
https://pennylane.readthedocs.io/en/latest/code/qml_operation.html

Reply:
But not too high priority, if anything a sentence or two

Reply:
Hmm, maybe, although we don't discuss other classes like `Tensor` and `Channel` here, so I'm wondering if we should make an exception for `Projection`?